packages/bierzo-wallet/src/logic/transactions/types/BwRegisterUsernameTx/ui/TransactionRow/index.tsx

```diff
@@ -0,0 +1,79 @@
+import { RegisterUsernameTx } from "@iov/bns";
+import { makeStyles, Theme } from "@material-ui/core";
+import { useTheme } from "@material-ui/styles";
+import { Block, CircleImage, Hairline, Image, Typography, useOpen } from "medulas-react-components";
+import * as React from "react";
+
+import toAddress from "../../../../../../routes/transactions/assets/toAddress.svg";
+import { getBorderColor } from "../../../../../../theme/css";
+import { formatDate, formatTime } from "../../../../../../utils/date";
+import { ProcessedTx } from "../../../../types/BwParser";
+import dropdownArrow from "../assets/dropdownArrow.svg";
+import dropdownArrowClose from "../assets/dropdownArrowClose.svg";
+import SendTxDetails from "./Details";
+
+const useStyles = makeStyles({
+  cell: {
+    flex: "1 0 50px",
+  },
+});
+
+interface Props {
+  readonly sendTx: ProcessedTx<RegisterUsernameTx>;
```

Comment:
`sendTx` -> `registerUsernameTx` or something like that

Reply:
I want to unify names in transaction templates so simply `tx` should be ok.

Reply:
Fine for me. Just `sendTx` does not make sense